### DISCUSSION TITLE
Cider's Spatial Audio 8D Audio or Headtracking

### DISCUSSION THREAD

**Discussion by bemwy** [ciderapp/Cider#996] [Rejected Features]
Could you add 8D audio to the spatialization audio effect? Also is headtracking with a gyro enabled headphone device possible for spatialization? (i.e. AirPods)

**Comment by cryptofyre** [comment]
No. Window's itself doesn't even allow access to hardware like that outside of VR devices let alone Apple and closed source hardware.
